packages/cli/src/commands/create-plugin/createPlugin.ts

```diff
@@ -343,16 +344,39 @@ const createPlugin = async () => {
         return true;
       },
     },
+    {
+      type: 'input',
+      name: 'owner',
+      message: chalk.blue(
+        'Enter the owner(s) of the plugin. If specified, this will be added to CODEOWNERS for the plugin path. [optional]',
+      ),
+      validate: (value: any) => {
+        if (!value) {
+          return true;
+        }
+
+        const ownerIds = parseOwnerIds(value);
+        if (!ownerIds) {
+          return chalk.red(
+            'The owner must be a space separated list of team names (e.g. @org/team-name), usernames (e.g. @username), or the email addresses of users (e.g. user@example.com).',
+          );
+        }
+
+        return true;
+      },
+    },
   ];
   const answers: Answers = await inquirer.prompt(questions);
 
   const rootDir = realpathSync(process.cwd());
+  const codeownersPath = path.join(rootDir, '.github', 'CODEOWNERS');
```

Comment:
We should search in all possible paths imo, `CODEOWNERS`, `docs/CODEOWNERS`, `.github/CODEOWNERS`, `.gitlab/CODEOWNERS`.

If you can be arsed, can also leave it for now 😅

Reply:
Ya sure, I thought I might